.env.docker

```
@@ -102,17 +102,18 @@ GENERATED_USERS_BATCH_SIZE=100
 
 AWS_ACCESS_KEY_ID=test
 AWS_SECRET_ACCESS_KEY=test
-AWS_REGION=us-east1
+AWS_REGION=eu-west-1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)